client/app/app.js

```diff
@@ -84,6 +84,19 @@ angular.module('pmsiplan', [
             },
             primaryKey: '_id'
         });
+
+        AngularDataStoreProvider.addModel({
+            name: 'histo',
+            fields: {
+                _id: {type: 'integer'},
+                resource: {type: 'string'},
+                action: {type: 'string'},
+                date: {type: 'date'},
```

Comment:
The equivalent date attribute in `https://github.com/pmsipilot/pmsiplan/blob/master/server/config/resources.js#L69` is a string. I think you can change it to the Date type even if it will affect the database format.